cot-cli/src/migration_generator.rs

```diff
@@ -1728,6 +1764,125 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_make_remove_model_operation() {
```

Comment:
For consistency sake 
```suggestion
    fn make_remove_model_operation() {
```